tests/core/test_dataset_filters.py

```diff
@@ -780,8 +780,8 @@ def test_compute_cell_sizes_composite(composite):
     assert output.n_blocks == composite.n_blocks
 
 
-def test_cell_centers(datasets):
-    for dataset in datasets:
+def test_cell_centers(datasets, airplane):
```

Comment:
The `datasets` fixture should probably be updated to include a `PointSet` dataset. This can be a separate PR as it will likely cause failures (i.e. discover bugs) with other filters.